README.md

```diff
@@ -30,7 +30,7 @@ your answer to this markdown file.
 
 ---------------------------------------------------------
 
-Since an invariant is to prove our function correct, an appropriate invariant would be an array containing all of the values of the fibonacci sequence up to that point, so for example fib(7) the invariant would be [0,1,1,2,3,5,8]
+Since an invariant is to prove our function correct, an appropriate invariant would be an array containing all of the values of the fibonacci sequence up to but not including some index $n$ where the array is [0,1,2,. . ., n-1], so for example fib(7) the invariant would be [0,1,1,2,3,5,8] with indices of [0,1,2,3,4,5,6].
```

Comment:
Your invariant must say something about a particular iteration, so it can't be any index.